funnel/explara.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+import requests
+
+__all__ = ['ExplaraAPI']
+
+
+class ExplaraAPI(object):
+    # explara_api = ExplaraAPI({'access_token'}: app.config.get('EXPLARA_ACCESS_TOKEN'))
+    # tickets = explara_api.get_tickets(app.config.get('EXPLARA_EVENT_IDS').get('rootconf-2015'))
+    def __init__(self, config):
+        self.access_token = config.get('access_token')
+
+    def get_orders(self, explara_event_id):
+        headers = {'Authorization': 'Bearer {0}'.format(self.access_token)}
+        attendee_list_url = 'https://www.explara.com/api/e/attendee-list'
+        ticket_orders = []
+        completed = False
+        from_record = 0
+        to_record = 50
+        while not completed:
+            payload = {'eventId': explara_event_id, 'fromRecord': from_record, 'toRecord': to_record}
+            attendee_response = requests.post(attendee_list_url, headers=headers, data=payload).json()
+            if not attendee_response.get('attendee'):
+                completed = True
+            elif isinstance(attendee_response.get('attendee'), list):
+                ticket_orders.append([order for order in attendee_response.get('attendee')])
+            # after the first batch, subsequent batches are dicts with batch no. as key.
+            elif isinstance(attendee_response.get('attendee'), dict):
+                ticket_orders.append([order for id, order in attendee_response.get('attendee').iteritems()])
```

Comment:
`id` is a builtin function and this line reassigns `id` to the last id in the list. Not good. Use `k, v` or some other variable name. Also, just `.items()` should do. `iteritems()` is deprecated in Python 3 and its behaviour is now the default.